concepts/functions/about.md

```diff
@@ -0,0 +1,153 @@
+# About
+
+A function allows to group code into a reusable unit.
+There are multiple ways to define functions in JavaScript.
+Here we will look at function declarations and function expressions.
+Other possibilities like [arrow functions][concept-arrow-functions] will be covered in other concepts.
+
+## Function Declaration
+
+The standard way of defining a function in JavaScript is a function declaration, also called function definition or function statement.
+
+It consists of the `function` keyword, the name of the function and a comma-separated list of parameters in round brackets.
```

Comment:
```suggestion
It consists of the `function` keyword, the name of the function, and a comma-separated list of parameters in round brackets.
```

I think we did decide on Oxford, but idk.

Reply:
fixed